apis/config/types_landscaper_config.go

```diff
@@ -67,3 +70,8 @@ type OCICacheConfiguration struct {
 	// +optional
 	Path string `json:"path"`
 }
+
+// MetricsConfiguration allows to configure how metrics are exposed
+type MetricsConfiguration struct {
+	Port string `json:"port"`
```

Comment:
i think it would be more natural to use a number here?
as in the k8s specs: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L1836